specification/electronic-prescription-service-api-original.yaml

```diff
@@ -0,0 +1,1111 @@
+# This is an OpenAPI Specification (https://swagger.io/specification/)
+# for the Electronic Prescription Service (EPS) API
+# owned by NHS Digital (https://digital.nhs.uk/)
+openapi: 3.0.0
```

Comment:
do we want to delete this file? we can use git history to track old versions!

Reply:
deleted